stl/inc/format

```diff
@@ -1300,8 +1329,13 @@ struct _Format_handler {
         if (_Begin == _End || *_Begin != '}') {
             throw format_error("Missing '}' in format string.");
         }
-        // TODO: implement format spec dispatching.
-        _STL_INTERNAL_CHECK(false);
+        _Ctx.advance_to(visit_format_arg(
+            _Arg_formatter<_OutputIt, _CharT>{
+                ._Ctx       = &_Ctx,

```

Comment:
All three of these expressions have types parameterized on a user-provided type, so `&` is hijackable. We need to `_STD addressof` instead of `&`.

Reply:
motivation to use references intensifies.gif

Reply:
> motivation to use references intensifies.gif

lol - had the same thought here.